Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -553,8 +555,8 @@ extension ActivityStreamPanel {
                                                                                     toApplication: UIApplication.shared)
                 site.setBookmarked(true)
             })
+            self.telemetry.reportEvent(.AddBookmark, source: pingSource, position: index)
```

Comment:
This line needs to go into the block above - similar to the `.RemoveBookmark` case.